config/crd/bases/anywhere.eks.amazonaws.com_cloudstackmachineconfigs.yaml

```diff
@@ -38,6 +38,11 @@ spec:
             description: CloudStackMachineConfigSpec defines the desired state of
               CloudStackMachineConfig
             properties:
+              affinity:
+                description: Defaults to `no`. Can be `pro` or `anti`. If set to `pro`
```

Comment:
`no`, `pro`, and `anti` are the only options that is provided by CAPC?

Reply:
> `no`, `pro`, and `anti` are the only options that is provided by CAPC?

yes.